docs/gettingstarted.md

```diff
@@ -271,6 +271,41 @@ $ ./torq.sh start rdb1 -extras -T 60 -w 4000
 $ ./torq.sh start sort1 -extras -s -3
 ```
 
+<a name="profiler">
+
+Using the Code Profiler with torq.sh
+------------------------------------
+
+KDB 4.0 includes an experimental built-in call-stack snapshot primitive that allows
+building a sampling profiler.  The profiler uses the new function [`.Q.prf0`](https://code.kx.com/q/ref/dotq/#qprf0-code-profiler). 
+As of now, this function has the following requirements:
+
+- Must be running KDB+ 4.0 or higher
```

Comment:
Can you just link to these requirements rather than changing them. Otherwise if kx update them, our documentation will be out of date